src/util.h

```diff
@@ -6,6 +6,16 @@
 #define NAPI_VERSION 1
 #include <node_api.h>
 
+#if __has_include(<sys/extattr.h>)
+#   include <sys/types.h>
+#   include <sys/extattr.h>
+#   include <sys/errno.h>
+#   define BSD_OS 1
```

Comment:
This is a poor choice of macro name.  For one thing, "BSD/OS" was the name of a now-defunct proprietary operating system from Wind River.  For another, it implies that all BSDs share a common interface.  However, OpenBSD does not.

Reply:
Hi Alan,
I changes poor macro name. This is feature C++17

Reply:
I reverted standart macro for detect operating system.